content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -13,16 +157,17 @@ operation:
   (credentials, private locations, etc). The local config file can be edited by
   hand or with the command `dvc config --local`.
 
-- `.dvc/cache`: The [cache directory](#structure-of-cache-directory) will store
-  your data. The data files and directories in the <abbr>workspace</abbr> will
-  only contain links to the data files in the cache. (Refer to
+- `.dvc/cache`: The <abbr>cache</abbr> directory will store your data in a
+  special [structure](#structure-of-cache-directory). The data files and
+  directories in the <abbr>workspace</abbr> will only contain links to the data
+  files in the cache. (Refer to
   [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization). See
   `dvc config cache` for related configuration options.
 
   > Note that DVC includes the cache directory in `.gitignore` during
   > initialization. No data tracked by DVC will ever be pushed to the Git
-  > repository, only [DVC-files](/doc/user-guide/dvc-file-format) that are
-  > needed to download or reproduce them.
+  > repository, only [DVC-files](/doc/user-guide/dvc-files-and-directories) that
+  > are needed to download or reproduce them.
 
 - `.dvc/plots`: Directory for
   [Plot templates](/doc/command-reference/plots#plot-templates).
```

Comment:
either put `.` after all sentences or remove here

Reply:
These weren't changed in this PR but sure, fixed. Only a couple of these

Reply:
I think we introduced quite a few items in this PR also, and looks like all of them are still not consistent (def not a blocker)

Reply:
Thanks. I double checked all the bullets just now and I did miss just 2 more. Fixed in cd7bca05

Keep in mind that some of the incomplete sentences still get a period because a sentence is coming after and we need to separate them.